packages/terra-breakpoints/src/ActiveBreakpointProvider.jsx

```diff
@@ -23,10 +23,14 @@ class ActiveBreakpointProvider extends React.Component {
 
   componentDidMount() {
     window.addEventListener('resize', this.setActiveBreakpoint);
+
+    // This event is needed to handle mobile device rotation since the resize event is not triggered.
+    window.addEventListener('orientationchange', this.setActiveBreakpoint);
```

Comment:
In case anyone was wondering, and I'm not sure if this is even possible, but if both handlers were to execute, setActiveBreakpoint will consolidate the calls into a single update.

Reply:
While debugging I did see that `resize` is called if and after `orientationchange` is called so it's great that they are consolidated.